src/sentry/api/endpoints/organization_integration_request.py

```diff
@@ -31,6 +31,7 @@ def get_url(organization, provider_type, provider_slug):
                     "sentry_app": "sentry-apps",
                 }.get(provider_type),
                 provider_slug,
+                "/?referrer=request_email",
```

Comment:
@ceorourke I wouldn't expect you need to have the extra `/` here if you're doing a join on `/`.

Reply:
oof you're right